src/pkg/utils/utils.go

```diff
@@ -250,7 +250,7 @@ func GetContainerNamePrefixForImage(image string) (string, error) {
 
 func getDefaultImageForDistro(distro, release string) string {
 	if !IsDistroSupported(distro) {
-		distro = "fedora"
+		distro = distroDefault
```

Comment:
As far as I can make out, this change seems functionally correct (ie., it won't break anything for the user), but it can cause confusion for the person reading the code.

The problem is that we have two similar, but subtly different, concepts that are important to way containers, distros, images and releases are handled; and we haven't been careful about clearly naming and separating the variables.

The *default* values are those that are used when the `create`, `enter` and `run` commands were used without any option.  These values are picked to match the host operating system.

However, if there's no supported Toolbox image matching the host operating system, and no options were provided to the `create`, `enter` and `run` commands, then the *fallback* values are used as a last resort.  Currently, the *fallback* values are set to Fedora.

Here, the string literal `"fedora"` was meant as a *fallback*, not *default*, value, because we never fall back to the *default* under any circumstances.  We should fall back to the *fallback* values if the *default* is not supported.

However, since the variables are neither clearly named nor separated, replacing the string literal with `distroDefault` doesn't change the behaviour of the code, but does make things even more confusing for the reader.